scripts/dockerfiles/polkadot/polkadot_builder.Dockerfile

```diff
@@ -1,10 +1,10 @@
 # This is the build stage for Polkadot. Here we create the binary in a temporary image.
 FROM docker.io/paritytech/ci-linux:production as builder
-
+ARG BUILD_FLAGS=""
```

Comment:
```suggestion
ARG BUILD_FLAGS
```

Reply:
Isn't that also working?

Reply:
Yes, but the docs don't specify what will happen to the value in case the user doesn't pass the argument so I'd personally prefer to be explicit. <sup>The docs also say there should be a warning but there wasn't in my quick test</sup>